src/Castle.Core.Tests/Castle.Core.Tests-VS2017.csproj

```diff
@@ -1,22 +1,45 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+	
 	<Import Project="..\..\buildscripts\common.props"></Import>
+
+	<PropertyGroup>
+		<TargetFrameworks Condition="'$(IsOSX)'=='true'">netcoreapp1.1</TargetFrameworks>
+		<TargetFrameworks Condition="'$(IsLinux)'=='true'">netcoreapp1.1</TargetFrameworks>
+		<TargetFrameworks Condition="'$(IsWindows)'=='true'">net461;netcoreapp1.1</TargetFrameworks>
+	</PropertyGroup>
```

Comment:
Could we do this like below and get rid of having to pass in `OsName` in build.sh:

```xml
<PropertyGroup>
  <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
  <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks
</PropertyGroup>
```